scanpy/readwrite.py

```diff
@@ -286,7 +287,10 @@ def _read_v3_10x_h5(filename, *, start=None, dask: bool = False):
                     feature_types=dsets['feature_type'].astype(str),
                     genome=dsets['genome'].astype(str),
                 )
-            adata = AnnData(X, obs, var, dask=dask,)
+            if dask:
+                adata = AnnDataDask(X, obs, var)
+            else:
+                adata = AnnData(X, obs, var)
```

Comment:
The latest `anndata` has the daskified logic in a separate class instead of passing through a boolean flag in the object, though the reader functions in anndata and scanpy use the flag up-to object construction.